### PR TITLE
Add tagline, revise home page intro text, plus minor css tweaks

### PR DESCRIPTION
**Pull request by simonbaird**
Ref: [HACBS-2514](https://issues.redhat.com/browse/HACBS-2514)

### CODE REVIEW

File: website/content/posts/a-taste-of-policies.md [hunk 1]
```diff
@@ -9,6 +9,8 @@ post](https://enterprisecontract.dev/posts/introducing-the-enterprise-contract/)
 basic concepts of the Enterprise Contract. This time, we explore it further to showcase the usage of
 policies.
 
+<!--more-->
+
```
Comment (zregvart) [inline website/content/posts/a-taste-of-policies.md:13]: The PR is not in draft, is this intentional?

**Review by zregvart — APPROVED** [review]

File: website/content/posts/a-taste-of-policies.md (hunk 1 above)
Comment (simonbaird) [reply website/content/posts/a-taste-of-policies.md:13]: Yeah, this is a significant markup for Hugo: It's used as a delimiter to mark the end of the blog post excerpt.

Comment (simonbaird) [reply website/content/posts/a-taste-of-policies.md:13]: Notice shorter excerpt at https://7a33b64c.enterprise-contract.pages.dev/posts/ vs https://enterprisecontract.dev/posts/ .

**Review by robnester-rh — APPROVED** [review]
lgtm

**Pull request merged**